.github/workflows/zowe-cli-plugin.yml

```diff
@@ -70,7 +70,7 @@ jobs:
 

```

Comment:
Can we also update the `t1m0thyj/unlock-keyring@master` to `t1m0thyj/unlock-keyring@v1`